Druid/DRBytecodeGenerator.class.st

```diff
@@ -20,6 +20,14 @@ DRBytecodeGenerator >> createTempVector: temporariesList [
 	
 ]
 
+{ #category : 'API' }
+DRBytecodeGenerator >> createTempVector: temporariesList vectorName: vectorName [
+	
+	temporariesList isNotEmpty ifTrue: [
```

Comment:
Probably we prefer this if outside of this method (in the caller). If not could be weird to call this method and do not create the tempVector (lier method)